types/module/configurator.go

```diff
@@ -47,12 +46,14 @@ type configurator struct {
 
 // NewConfigurator returns a new Configurator instance
 func NewConfigurator(cdc codec.Codec, msgServer grpc.Server, queryServer grpc.Server) Configurator {
-	return configurator{
+	c := configurator{
```

Comment:
```suggestion
	return configurator{
```
?

Reply:
leftover from the removal of validation part.